src/frontend/web/src/ratingpage/Ratingpage.scss

```diff
@@ -0,0 +1,147 @@
+.layout {
```

Comment:
Again - commenting here because I couldn't find for to comment on binaries - please move the png files from this directory to the `public` folder. They belong there :)

Reply:
Apparently media placed in the "public" folder don’t get pre-processed but copied as is to the build. To avoid this, the [docs](https://create-react-app.dev/docs/using-the-public-folder/#adding-assets-outside-of-the-module-system) recommend using relative imports, and relative imports outside of the "src" folder are not supported.
So maybe placing the png files in the "assets" folder I’ve mentioned [here](https://github.com/saifun/check_mate/pull/17#discussion_r400314357) would be a better idea? What do you think?

Reply:
On production, pictures and other static media should are brought to the user using NginX (read about it) and not in the same path as the website pages.
You could look for an example in React's code - they put some pics in the public folder and use it in the code.

Reply:
Got you, done!